### PR TITLE
Enforced uniqueness of medicare number

### PR DESCRIPTION
**Pull request by Vegemash**

### CODE REVIEW

File: protected/tests/unit/models/AUSPatientTest.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Created by PhpStorm.
```
Comment (marshl) [inline protected/tests/unit/models/AUSPatientTest.php]: Please try to remove, or update these messages

File: protected/tests/unit/models/AUSPatientTest.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+ * Date: 22/09/17
+ * Time: 2:11 PM
+ */
+
+class AUS_PatientTest extends CDbTestCase
+{
+    protected $fixtures = array('patient' => 'Patient');
+    public function setUp()
+    {
+        parent::setUp();
```
Comment (marshl) [inline protected/tests/unit/models/AUSPatientTest.php]: You might not need these if they are just calling the parents, although they might be useful later...

File: protected/tests/unit/models/AUSPatientTest.php [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        $patient_dup = new Patient;
+        $patient_dup->attributes = $this->patient['patient1'];
+
+        //change the things we don't want it to fail on
+        $patient_dup->id = NULL;
+        $patient_dup->pas_key = NULL;
+        $patient_dup->hos_num = 65465465;
+
+        //fail to save with duplication
+        $this->assertFalse($patient_dup->save());
```
Comment (marshl) [inline protected/tests/unit/models/AUSPatientTest.php]: You might want to check exactly which field the save failed on. $patient_dup->getError('nhs_num') could be useful

File: protected/tests/unit/models/AUSPatientTest.php (hunk 2 above)
Comment (apigram) [reply protected/tests/unit/models/AUSPatientTest.php]: I'd say leave them in for the time being. There is every chance we'll need to add more to these functions as more test cases are added.

**Review by apigram — APPROVED** [review]

**Review by marshl — APPROVED** [review]

**Pull request merged**